test/evp_extra_test.c

```diff
@@ -4717,6 +4721,161 @@ static int test_ecx_not_private_key(int tst)
+                "Ed25519",
+                "provider=default",
+                rfc8032_test_vectors[i].sk,
+                32);
+
+        pkey_pk = EVP_PKEY_new_raw_public_key_ex(
+                testctx,
+                "Ed25519",
+                "provider=default",
+                rfc8032_test_vectors[i].pk,
+                32);
+
+    } else {
+
+        expected_sig_length = Ed448_SIGNATURE_LENGTH;
+        pkey_sk = EVP_PKEY_new_raw_private_key_ex(
+                testctx,
+                "Ed448",
+                "provider=default",
+                rfc8032_test_vectors[i].sk,
+                57);
+
+        pkey_pk = EVP_PKEY_new_raw_public_key_ex(
+                testctx,
+                "Ed448",
+                "provider=default",
+                rfc8032_test_vectors[i].pk,
+                57);
+
+    }
+
+    if (pkey_sk == NULL) {
+        printf("failed (%d)\n", __LINE__);
+        goto err;
+    }
+
+    if (pkey_pk == NULL) {
+        printf("failed (%d)\n", __LINE__);
+        goto err;
+    }
+
+    md_ctx = EVP_MD_CTX_new();
+
+    params[0].key = "instance";
+    params[0].data_type = OSSL_PARAM_UTF8_STRING;
+    params[0].data = (char *)rfc8032_test_vectors[i].algorithm;
+    params[0].data_size = rfc8032_test_vectors[i].algorithm_length;
+    params[0].return_size = (size_t)(-1);
+
+    params[1].key = "context-string";
+    params[1].data_type = OSSL_PARAM_OCTET_STRING;
+    params[1].data = (unsigned char *)rfc8032_test_vectors[i].context;
+    params[1].data_size = rfc8032_test_vectors[i].context_length;
+    params[1].return_size = (size_t)(-1);
+
+    params[2].key = NULL;
+    params[2].data_type = 0;
+    params[2].data = NULL;
+    params[2].data_size = 0;
+    params[2].return_size = 0;
+
+    /**** sign ****/
+
+    rc = EVP_DigestSignInit_ex(md_ctx, NULL,
+            NULL, testctx,
+            NULL, pkey_sk,
+            params);
+    if (rc==0) {
+        printf("failed (%d)\n", __LINE__);
+        goto err;
+    }
+
+    sig_length = sizeof(sig);
+    rc = EVP_DigestSign(
+            md_ctx,
+            sig,
+            &sig_length,
+            rfc8032_test_vectors[i].msg,
+            rfc8032_test_vectors[i].msg_length);
+    if (rc==0) {
+        printf("failed (%d)\n", __LINE__);
+        goto err;
+    }
+
+    if (sig_length != expected_sig_length
+            || memcmp(sig, rfc8032_test_vectors[i].sig, sig_length) != 0) {
+        printf("failed (%d)\n", __LINE__);
+        goto err;
+    }
+
+    /**** verify ****/
+
+    rc = EVP_DigestVerifyInit_ex(md_ctx, NULL,
+            NULL, testctx,
+            NULL, pkey_pk,
+            params);
+    if (rc==0) {
+        printf("failed (%d)\n", __LINE__);
+        goto err;
+    }
+
+    rc = EVP_DigestVerify(
+            md_ctx,
+            sig,
+            sig_length,
+            rfc8032_test_vectors[i].msg,
+            rfc8032_test_vectors[i].msg_length);
+    if (rc==0) {
+        printf("failed (%d)\n", __LINE__);
```

Comment:
Remove printf and use TEST_error or similar here

Reply:
printf's are gone and nits picked.  thx!